.travis.yml

```diff
@@ -15,12 +17,12 @@ cache:
     - $HOME/.composer/cache
     - $HOME/.cache/bin
 matrix:
-  exclude:
-    - php: 5.6
-      env: TEST_SUITE=static_phpcs
-    - php: 5.6
-      env: TEST_SUITE=static_annotation
```

Comment:
This was made intentionally. In static tests application code is not executed thus there is no need to run them on different versions of PHP.

In such a way you would check that PHPCS works the same way under PHP5.6, PHP7 and other versions.

Please exclude all variants but PHP7 as the fastest one.